eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php

```diff
@@ -1215,6 +1215,55 @@ public function testRefreshSystemUrlAliasesForContentsWithUpdatedContentTypes()
         $this->assertUrlIsCurrent('/DE-Short-Name/DE-Nested-Short-Name', $nestedFolderLocation->id);
     }
 
+    /**
+     * Test that created non-latin aliases are non-empty and unique.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\ForbiddenException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function testCreateNonLatinNonEmptyUniqueAliases()
+    {
+        $urlAliasService = $this->getRepository()->getURLAliasService();
+        $locationService = $this->getRepository()->getLocationService();
```

Comment:
```php
        $repository = $this->getRepository();
        $urlAliasService = $repository->getURLAliasService();
        $locationService = $repository->getLocationService();
```
Otherwise you're initializing database twice :upside_down_face: